Jenkinsfile

```diff
@@ -37,65 +37,88 @@ void unittests(String version) {
                         find . -name *.html
                         find . -name *.xml
                     """
-                }
-                catch(ex) {
-                    throw ex
-                }
-                finally {
-                    if (fileExists ('test-reports/coverage/index.html')) {
-                        publishHTML(
-                            [allowMissing: true,
-                             alwaysLinkToLastBuild: true,
-                             keepAll     : true,
-                             reportDir   : 'test-reports/coverage/',
-                             reportFiles : 'index.html',
-                             reportName  : "Coverage${version}",
-                             reportTitles: "Coverage${version}"])
-                    }
-                    step([$class: 'JUnitResultArchiver', testResults: 'test-reports/junit/*.xml'])
-                }
-            }
-        }
-    }
+				}
+				catch(ex) {
+					throw ex
+				}
+				finally {
+					if (fileExists('test-reports/coverage/index.html')) {
+						publishHTML([allowMissing: true, alwaysLinkToLastBuild: true, keepAll: true, reportDir: 'test-reports/coverage/', reportFiles: 'index.html', reportName: "Coverage${version}", reportTitles: "Coverage${version}"])
+					}
+					step([$class: 'JUnitResultArchiver', testResults: 'test-reports/junit/*.xml'])
+				}
+			}
+		}
+	}
 }
 
 try {
-    Map tasks = [ : ]
-
-    pythonVersion.each {
-        version ->
-            tasks[version] = { unittests(version) }
-    }
-
-    tasks['snyk'] = {
-        stage('snyk') {
-            Snyk snyk = new Snyk(this, bparams)
-            snyk.execute()
-        }
-    }
-
-    tasks['sonarqube'] = {
-        stage('sonarqube') {
-            SonarQube.execute(this, bparams)
-        }
-    }
-
-    tasks['nexusiq'] = {
-        stage('nexusiq') {
-            Nexusiq.execute(this, bparams)
-        }
-    }
-
-    parallel(tasks)
-
-    common.setResultIfNotSet(Constants.JSUCCESS)
-} 
-catch (ex) {
-    common.logException(ex)
-    common.setResultAbortedOrFailure()
-    throw ex
-} 
-finally {
-    common.setResultIfNotSet(Constants.JFAILURE)
-    buildsCommon.notifyPostBuild(bparams)
+	Map tasks = [: ]
+
+	pythonVersion.each {
+		version ->tasks[version] = {
+			unittests(version)
+		}
+	}
+
+	tasks['snyk'] = {
+		stage('snyk') {
+			Snyk snyk = new Snyk(this, bparams)
+			snyk.execute()
+		}
+	}
+
+	tasks['sonarqube'] = {
+		stage('sonarqube') {
+			SonarQube.execute(this, bparams)
+		}
+	}
+
+	tasks['nexusiq'] = {
+		stage('nexusiq') {
+			Nexusiq.execute(this, bparams)
+		}
+	}
+
+	tasks['runPylint'] = {
+		stage('runPylint') {
+			node(Constants.DOCKERNODE) {
+				withContainer(image: "python:3.6-buster", registry: '', inside: '-u root') {
```

Comment:
You are in a new docker node. There is no source code unless you checkout the code.

Reply:
Added as suggested